src/screens/MainApp/MyDiagnoses/screens/SolutionRequest/index.js

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from 'react'
+import { TextInput, Alert, ScrollView, View, BackHandler } from 'react-native'
+import { scale, verticalScale, moderateScale } from 'react-native-size-matters/extend'
+import { SvgXml } from 'react-native-svg'
+import { BoxShadow } from 'react-native-shadow'
+import { PrimaryButton, Body, Description, Subtitle } from '~/components'
+import * as AnalyticsService from '~/analyticsService'
+import * as DatabaseService from '~/databaseService'
+import Background from '~/components/Background'
+import { theme } from '~/constants'
+import styles from './styles'
+import logo from '~/assets/images/SolutionRequest/logo.svg'
+
+const SolutionRequest = ({ navigation }) => {
+  AnalyticsService.setCurrentScreenName('Solution Request')
+
+  const [description, setDescription] = useState('')
+  const [inputEnabled, setInputEnabled] = useState(true)
+  const [uploading, setUploading] = useState(false)
+
+  const diagnose = navigation.state.params.diagnose
+
+  const handleInput = (text) => {
+    setDescription(text)
+  }
+
+  const handleUpload = async () => {
+    setInputEnabled(false)
+    setUploading(true)
+    try {
+      await DatabaseService.addDiagnoseResponse(diagnose.id, description)
+      await DatabaseService.setDiagnoseSolvedMark(diagnose.id, true)
+
+      setDescription('')
+      setUploading(false)
+      setInputEnabled(true)
+
+      navigation.navigate('FinishScreen')
+    } catch (error) {
+      console.log(error)
+      Alert.alert(
+        'Error',
+        'No se pudo realizar la operación. Por favor, intente nuevamente más tarde.'
+      )
+      setUploading(false)
+      setInputEnabled(true)
```

Comment:
Please remove the `console.log` and a blank before `setUploading` and `setInputEnabled`